tests/test_questions.py

```diff
@@ -148,6 +148,25 @@ def test_get_question_ids_by_type(self):
         assert self.question().get_question_ids(type='text') == ['example']
 
 
+class TestBoolean(QuestionTest):
+    def question(self, **kwargs):
+        data = {
+            "id": "example",
+            "type": "boolean"
+        }
+        data.update(kwargs)
+
+        return ContentQuestion(data)
+
+    def test_followup_values(self):
```

Comment:
The `followup:` YAML format is the same for both, and that's the only thing this property is using, so it's the same for radios and checkboxes.